docs/examples/simple_load_and_predict.py

```diff
@@ -18,20 +16,4 @@
 trees = m.predict_tile(image=image, patch_size=3000, patch_overlap=0)
 # Filter out low-confidence detections
 trees = trees[trees.score > 0.3]
-
-# Convert the tree detections to Supervision format for visualization
-sv_detections = visualize.convert_to_sv_format(trees)
-
-# Create a bounding box annotator
-bounding_box_annotator = sv.BoxAnnotator()
-
-# Annotate the image with bounding boxes
-annotated_frame = bounding_box_annotator.annotate(
-    scene=image,
-    detections=sv_detections
-)
-
-# Display the annotated image using Matplotlib
-plt.imshow(annotated_frame)
-plt.axis('off')  # Hide axes for a cleaner look
-plt.show()
+visualize.plot_results(trees, root_dir=os.path.dirname(image_path))
```

Comment:
Suggest removing the following three lines above:

```python
# Load the image
image_path = "<path_to_image>"
image = cv2.imread(image_path)
```

Using `predict_tile()` with `raster_path` and then dropping the use of `root_dir` here (and therefore the `import os`).

This feels like the simplest most standard application, while manually loading with `cv2` and the setting `root_dir` seems like a more expert user in less need to examples.